sniffles/test/data_files/rules/test_rules1.rules

```diff
@@ -8,4 +8,4 @@ alert tcp $EXTERNAL_NET any -> $HOME_NET 8081 (msg:"test1-6"; flow:to_server,est
 alert tcp $EXTERNAL_NET any -> $HOME_NET $HTTP_PORTS (msg:"test1-7"; flow:to_server,established; content:"/struts"; nocase; http_uri; content:"|25|252f"; nocase; http_raw_uri; classtype:attempted-recon; sid:7;)
 alert tcp $EXTERNAL_NET any -> $HOME_NET $HTTP_PORTS (msg:"test1-8"; flow:to_server,established; content:"Proxy-Connection: Keep-Alive"; http_header; content:"Transfer-Encoding: chunked|0D 0A|Content-Length: 40334"; fast_pattern:only; http_header; content:".dll"; http_uri; classtype:attempted-admin; sid:8;)
 alert tcp $EXTERNAL_NET any -> $HOME_NET 8080 (msg:"test1-9"; flow:to_server,established; content:"%2E%2E/"; fast_pattern:only; content:"%2E%2E/"; nocase; http_raw_uri; pcre:"/\/(\\|%5C)%2E%2E\//"; sid:9;)
-alert tcp $EXTERNAL_NET any -> $HOME_NET $HTTP_PORTS (msg:"test1-10"; flow:to_server,established; content:"POST"; http_method; content:"=|27|"; http_client_body; content:"new"; nocase; http_client_body; pcre:"/new(%20)?(javax?|org|com)/i"; classtype:attempted-admin; sid:10;)
\ No newline at end of file
+alert tcp $EXTERNAL_NET any -> $HOME_NET $HTTP_PORTS (msg:"test1-10"; flow:to_server,established; content:"POST"; http_method; content:"=|27|"; http_client_body; content:"new"; nocase; http_client_body; pcre:"/new(%20)?(javax?|org|com)/i"; classtype:attempted-admin; sid:10;)
```

Comment:
Thank you. This is the kind of feedbacks I wanted. The intent was not to add a fancy new feature to sniffles, but to let @mskpyn understand the code structure and figure out how to extend sniffles for more functionalities.

As for the goal of these changes, what would you suggest to create a mixture of a large number of HTTP streams, of which the payloads may be described by numerous regular expressions or other models? What would be the best way to create a group of random streams without defining each stream individually?